soroban-env-common/env.json

```diff
@@ -1447,6 +1447,48 @@
                     ],
                     "return": "AddressObject",
                     "docs":  "Get the id of the Stellar Asset contract corresponding to the provided asset without creating the instance. `serialized_asset` is `stellar::Asset` XDR serialized to bytes format. Returns the address of the would-be asset contract."
+                },
+                {
+                    "export": "c",
+                    "name": "extend_contract_instance_ttl",
+                    "args": [
+                        {
+                            "name": "contract",
+                            "type": "AddressObject"
+                        },
+                        {
+                            "name": "threshold",
+                            "type": "U32Val"
+                        },
+                        {
+                            "name": "extend_to",
+                            "type": "U32Val"
+                        }
+                    ],
+                    "return": "Void",
+                    "docs": "If the TTL for the provided contract instance is below `threshold` ledgers, extend `live_until_ledger_seq` such that TTL == `extend_to`, where TTL is defined as live_until_ledger_seq - current ledger.",
+                    "min_supported_protocol": 21
+                },
+                {
+                    "export": "d",
+                    "name": "extend_contract_code_ttl",
+                    "args": [
+                        {
+                            "name": "contract",
+                            "type": "AddressObject"
+                        },
+                        {
+                            "name": "threshold",
+                            "type": "U32Val"
+                        },
+                        {
+                            "name": "extend_to",
+                            "type": "U32Val"
+                        }
+                    ],
+                    "return": "Void",
+                    "docs": "If the TTL for the provided contract code (if applicable) is below `threshold` ledgers, extend `live_until_ledger_seq` such that TTL == `extend_to`, where TTL is defined as live_until_ledger_seq - current ledger.",
```

Comment:
nit: "contract's" - with the current wording I would expect Wasm hash as an argument.

Reply:
Fixed